docs/layers.md

```diff
@@ -242,6 +242,8 @@ Landuse polygons from OpenStreetMap representing parks, forests, residential, co
 
 Zooms 4 and 5, 6 and 7 includes a mix of Natural Earth `urban_area` (zooms 0-9 only) features and OpenStreetMap data for `national_park`, `protected_area`, and `nature_reserve` only. After that more more feature kinds are included, and they have a richer set of properties including `sport`, `religion`, `surface`, `attraction`, `zoo`, and `natural`. Feature selection is filtered per zoom until zoom 15.
 
+At mid-zooms, between 9-12, some landuse polygons are merged to reduce payload size. To facilitate this, the name of the landuse area may be dropped for small polygons. When polygons are merged, the original `id` properties are dropped, and the `area` is re-calculated for the new size.
```

Comment:
You're correct, zoom 4.



> On Aug 17, 2016, at 07:32, Matt Amos <notifications@github.com> wrote:
> 
> In docs/layers.md:
> 
> > @@ -242,6 +242,8 @@ Landuse polygons from OpenStreetMap representing parks, forests, residential, co
> >  
> >  Zooms 4 and 5, 6 and 7 includes a mix of Natural Earth `urban_area` (zooms 0-9 only) features and OpenStreetMap data for `national_park`, `protected_area`, and `nature_reserve` only. After that more more feature kinds are included, and they have a richer set of properties including `sport`, `religion`, `surface`, `attraction`, `zoo`, and `natural`. Feature selection is filtered per zoom until zoom 15.
> >  
> > +At mid-zooms, between 9-12, some landuse polygons are merged to reduce payload size. To facilitate this, the name of the landuse area may be dropped for small polygons. When polygons are merged, the original `id` properties are dropped, and the `area` is re-calculated for the new size.
> I can see that mz_landuse_min_zoom might be as low as 2, but the queries only appear to kick in at zoom 4.
> 
> Have I overlooked some way we can get polygons in landuse at z<4?
> 
> ―
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>